packages/labs/context/package.json

```diff
@@ -38,8 +38,8 @@
     "clean": "rm -rf index.* lib/ development/ test/ *.tsbuildinfo",
     "dev": "scripts/dev.sh",
     "test": "npm run test:dev && npm run test:prod",
-    "test:dev": "MODE=dev node ../../tests/run-web-tests.js \"development/**/*_test.js\"",
-    "test:prod": "MODE=prod node ../../tests/run-web-tests.js \"development/**/*_test.js\"",
+    "test:dev": "MODE=dev node ../../tests/run-web-tests.js \"development/**/*_test.js\" --config ../tests/web-test-runner.config.js",
+    "test:prod": "MODE=prod node ../../tests/run-web-tests.js \"development/**/*_test.js\" --config ../tests/web-test-runner.config.js",
```

Comment:
This needs two go up two directories, because `context` is in labs.

I.e. `../../tests/web-test-runner.config.js`